test/test_helper.exs

```diff
@@ -66,25 +67,40 @@ defmodule TestHelper do
     :ok
   end
 
-  def edoc_to_chunk(module) do
-    source_path = module.module_info(:compile)[:source]
-    beam_path = :code.which(module)
-    dir = :filename.dirname(source_path)
-    xml_path = '#{dir}/#{module}.xml'
-    chunk_path = '#{dir}/#{module}.chunk'
-
-    docgen_dir = :code.lib_dir(:erl_docgen)
-    cmd!("escript #{docgen_dir}/priv/bin/xml_from_edoc.escript -dir #{dir} #{source_path}")
-
-    :docgen_xml_to_chunk.main(["app", xml_path, beam_path, "", chunk_path])
-    docs_chunk = File.read!(chunk_path)
-    {:ok, ^module, chunks} = :beam_lib.all_chunks(beam_path)
-    {:ok, beam} = :beam_lib.build_module([{'Docs', docs_chunk} | chunks])
-    File.write!(beam_path, beam)
-  end
-
-  defp cmd!(command) do
-    0 = Mix.shell().cmd(command)
+  if Code.ensure_loaded?(:edoc_doclet_chunks) do
+    def edoc_to_chunk(module) do
+      source_path = module.module_info(:compile)[:source]
+      dir = :filename.dirname(source_path)
+
+      :ok =
+        :edoc.files([source_path],
+          preprocess: true,
+          doclet: :edoc_doclet_chunks,
+          layout: :edoc_layout_chunks,
+          dir: dir ++ '/doc'
+        )
+    end
+  else
+    def edoc_to_chunk(module) do
+      source_path = module.module_info(:compile)[:source]
+      beam_path = :code.which(module)
+      dir = :filename.dirname(source_path)
+      xml_path = '#{dir}/#{module}.xml'
+      chunk_path = '#{dir}/#{module}.chunk'
+
+      docgen_dir = :code.lib_dir(:erl_docgen)
```

Comment:
maybe use `:code.priv_dir(:erl_docgen)` instead of `:code.lib_dir(:erl_docgen)`?
because on 92 line you pass the `priv` folder path, make sense?

Reply:
fixed on master, thx!